ts-packages/web/src/app/(social)/threads/[id]/_components/space-create-modal/select-space-form.tsx

```
@@ -174,6 +177,59 @@ export default function SelectSpaceForm({ feed_id }: { feed_id: number }) {
+  const SpaceForm = useMemo(
+    () =>
+      function SpaceFormInner({
+        form,
+        selected,
+        onClick,
+      }: {
+        form: SpaceFormProps;
+        selected: boolean;
+        onClick: () => void;
+      }) {
+        const tt = useTranslations('SpaceForms');
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)